usermods/Tubes/node.h

```diff
@@ -273,7 +349,7 @@ class LightNode {
             // Execute the command
             auto valid = receiver->onCommand(
                 message->command,
-                &message->data
+                const_cast<uint8_t*>(message->data)
```

Comment:
This actually removes the const because the onCommand doesn't take a const.
I could go either way on this, and it kind of depends on who owns the memory.  Using const helps cast errors at compile that when modifying something that wasn't intentional.  But given the const gets cast away and the buffer is modify, I could see not forcing const.   That being said, if we make the network protocol more flexible with different messages, it might be beneficial to keep the const in there so we don't have accidental buffer overrides when messages are different sizes

Reply:
I agree that if we can actually get the messages to specialized methods that expect a certain size, and somehow the dispatcher can use const to enforce that, it'll be safer
Might be worth exploring this once we have more than one message (I mean, there is right now, but it's really only the status update message that gets used)